client/src/resources/map/MapModel.js

```diff
@@ -57,18 +56,17 @@ const mapDispatchToProps = dispatch => (
         method: 'GET',
         credentials: 'same-origin',
         headers: {
-          'Accept': 'application/json',
-          'Content-Type': 'application/json',
-        }
+          Accept: 'application/json',
+          ContentType: 'application/json',
```

Comment:
I believe these will throw an error on the server. Double-check. Better to have angry eslint than dysfunctional route. Same for all 'content-type' instances